app/client/cypress/integration/community/search.spec.js

```diff
@@ -8,5 +8,46 @@ describe('Community search', () => {
     cy.findByPlaceholderText('Search by address', { exact: false }).type(zip);
     cy.findByText('Go').click();
     cy.url().should('include', `/community/${zip}/overview`);
+    cy.findByText('Your Waters: What We Know').should('exist');
+  });
+
+  it('searching for gibberish displays an error', () => {
+    const search = 'jkljl';
+    cy.findByPlaceholderText('Search by address', { exact: false }).type(
+      search,
+    );
+    cy.findByText('Go').click();
+    cy.url().should('equal', `${window.location.origin}/community`);
+    cy.findByText('Data is not available for this location.', {
+      exact: false,
+    }).should('exist');
+  });
+
+  it('searching for a valid huc properly routes to the community overview page', () => {
+    const search = '020700100103';
+    cy.findByPlaceholderText('Search by address', { exact: false }).type(
+      search,
+    );
+    cy.findByText('Go').click();
+    cy.url().should('include', `/community/${search}/overview`);
+    cy.findByText('Your Waters: What We Know').should('exist');
+  });
+
+  it('Clicking the “Use My Location” button, with geolocation enabled, properly geolocates user and routes to the community overview page for the user’s location', () => {
+    cy.mockGeolocation();
+
+    cy.findByText('Use My Location').click();
+
+    cy.url().should('match', /community\/.*\/overview/);
+    cy.findByText('Your Waters: What We Know').should('exist');
+  });
+
+  it('Clicking the “Use My Location” button, with geolocation disabled, displays an error and does not route', () => {
+    cy.mockGeolocation(true);
+
+    cy.findByText('Use My Location').click();
+
+    cy.url().should('equal', `${window.location.origin}/community`);
+    cy.findByText('Error Getting Location').should('exist');
```

Comment:
Just tightening up whitespace to be consistent with other tests.

```suggestion
    cy.mockGeolocation(true);
    cy.findByText('Use My Location').click();
    cy.url().should('equal', `${window.location.origin}/community`);
    cy.findByText('Error Getting Location').should('exist');
```